City-of-Helsinki/berth-reservations
# VEN-997 | Email template generation

## Description :sparkles:

* Added `generate.py` script for combining messages and base templates
* Converted all existing templates from `TatuArvela/hel-email-templates` repository

## Issues :bug:
### Closes :no_good_woman:
**[VEN-997](https://helsinkisolutionoffice.atlassian.net/browse/VEN-997):** Convert email templates to a more plain solution

### Related :handshake:
**[VEN-981](https://helsinkisolutionoffice.atlassian.net/browse/VEN-981):** Create email template for additional service invoice
  * only Finnish version included

**[VEN-994](https://helsinkisolutionoffice.atlassian.net/browse/VEN-981):** Email template for "no berth to assign" case
  * only Finnish version included

## Testing :alembic:
### Automated tests :gear:️
None

### Manual testing :construction_worker_man:
1. Run `generate.py`
2. View files under `generated` in your browser
* Note that the Jinja tags may render incorrectly, they are only parsed and populated in rendered emails
